README.rst

```diff
@@ -194,10 +257,11 @@ Or run it directly:
 
 .. code:: bash
 
-    $ python -m httpbin.core [--port=PORT] [--host=HOST]
+    $ python -m httpbin.core [--port=PORT] [--host=HOST] [--enable-file-endpoint] [--disable-file-endpoint]
```

Comment:
how are these flags intended to be used?  I would think that if you don't set `--enable-file-endpoint`, then it would default to be disabled? (or vice versa)

What happens if you pass both?